Input/Input.module.less

```diff
@@ -59,8 +63,9 @@
 
 		.input,
 		.icon {
-			line-height: @agate-input-height;
+			font-family: @agate-base-font-family;
 			height: @agate-input-height;
+			line-height: @agate-input-line-height;
```

Comment:
This shouldn't have needed to change. The line-height here is set to the input height to ensure that the text and icon are always properly vertically centered in the field.

The font-family should _absolutely not change_ on this line. This applies to both `.input` and `.icon`. Icon has its own defined font family. If you're intending to have the input field use a different font than the rest of the skin, is above, on line 13 or 23.

That being said, I don't think you need to actually set the font-family in this component at all. It inherits safely from the root ThemeDecorator.